concent_api/command_line_tool/message_handler.py

```diff
@@ -43,14 +43,23 @@ def _print_message_info(message_info):
     print('-' * len(message_info) + '\n' + str(message_info) + '\n' + '-' * len(message_info))
 
 
+def print_response_info(response):
+    print('STATUS: {}'.format(response.status_code))
+    print('Content-Type: {}'.format(response.headers['Content-Type']))
```

Comment:
Please use `f` function to write parameters to string e.g:
```python
f`STATUS: {response.status_code}`
```
If we can we should use it because it looks better than `.format` at the end of string